scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -1032,13 +1032,26 @@ export function AuthoringDirective(
                     }
                 }
 
-                $rootScope.$broadcast('item:nextStage', {
-                    stage: stageList[stageIndex],
-                    itemId: $scope.item._id,
-                    selectedStage: selectedStage,
-                    selectedDesk: selectedDesk,
-                    item: $scope.item,
-                });
+                if ($location.$$path === '/workspace') {
```

Comment:
It'd be good to avoid URL-specific behavior. I assume your intention is to check whether next stage is available, and you are using URL as a metric, but it'd be better to do a direct check, by checking `scope.currentDesk` or something similar.

It's also not clear to me why `'/workspace'` is used. Judging from the ticket, it looks like sending works from monitoring, and doesn't work from search, so I'd expect a special case to be added to handle sending from search.

As Mark pointed out, `beforeSend` hooks need to be fired, which it wouldn't be in this case. I also agree on having the sending logic in one place, but it would be quite hard to add it to internal API, because internal API is just a plain JS file with exported stateless functions that doesn't have access to any angular state. Instead I suggest implementing this in `scope.send` function in `scripts/apps/authoring/authoring/directives/SendItem.tsx`(after `beforeSend` is ran).